bruinen/src/bruinen/langchain/parsers/sql_parser.py

```diff
@@ -97,13 +78,9 @@ def map_sql_types(type_var):
 
         metadata = MetaData()
         metadata.reflect(bind=engine)
-        sql_table = Table(
-            input_class_name.lower(), metadata, autoload=True, autoload_with=engine
-        )
+        sql_table = Table(input_class_name.lower(), metadata, autoload=True, autoload_with=engine)
 
-        template_table = "\nOnly use the following tables:\n" + str(
-            CreateTable(sql_table).compile(engine)
-        )
+        template_table = "\nOnly use the following tables:\n" + str(CreateTable(sql_table).compile(engine))
         # If the table isn't too large, would be better to also include example data
```

Comment:
An example row is certainly something we should consider.

Lets start a feature list and design doc for this, lets find the short-comings and improve on them